src/app/components/VisuallyHiddenText/README.md

```diff
@@ -0,0 +1,55 @@
+# visually-hidden-text - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/simorgh/badge.svg?targetFile=packages%2Fcomponents%2Fsimorgh-visually-hidden-text%2Fpackage.json)](https://snyk.io/test/github/bbc/simorgh?targetFile=packages%2Fcomponents%2Fsimorgh-visually-hidden-text%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/simorgh.svg?path=packages/components/simorgh-visually-hidden-text)](https://david-dm.org/bbc/simorgh?path=packages/components/simorgh-visually-hidden-text) [![peerDependencies Status](https://david-dm.org/bbc/simorgh/peer-status.svg?path=packages/components/simorgh-visually-hidden-text)](https://david-dm.org/bbc/simorgh?path=packages/components/simorgh-visually-hidden-text&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/simorgh/?path=/story/visuallyhiddentext--default) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/simorgh/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/#legacy/simorgh-visually-hidden-text/src.svg)](https://www.npmjs.com/package/#legacy/simorgh-visually-hidden-text/src) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md)
+
+## Description
+
+Adds non-visible text to the page, to be utilised by screen-readers and other assistive technology.
+
+## Installation
+
+`npm install #app/components/simorgh-visually-hidden-text/src`
+
+## Props
+
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| No props. |      |          |         |         |
+
+## Usage
+
+```tsx
+import VisuallyHiddenText from '#app/components/VisuallyHiddenText';
```

Comment:
We'll need to update the import path there - according to the guidelines we should now try to avoid using the # path going forward

```suggestion
import VisuallyHiddenText from 'src/app/components/VisuallyHiddenText';
```